cpu/cortexm_common/thread_arch.c

```diff
@@ -259,10 +259,12 @@ int thread_isr_stack_usage(void)
 {
     uint32_t *ptr = &_sstack;
 
+    /* cppcheck-suppress comparePointers */
     while(((*ptr) == STACK_CANARY_WORD) && (ptr < &_estack)) {
```

Comment:
```suggestion
    while (((*ptr) == STACK_CANARY_WORD) && (ptr < &_estack)) {
```

Reply:
This should be fixed when we touch the files anyway.

Reply:
Done